zipa/resource.py

```diff
@@ -74,29 +75,36 @@ def _prepare_data(self, **kwargs):
         else:
             return kwargs
 
-    def _prepare_entity(self, response):
+    @staticmethod
+    def default_response_handler(response):
```

Comment:
Maybe extract it into a different class.
It can contains methods like:
`parse_response`
`has_error`
`parse_error_message`.

But maybe is too complex for a simple usecase.